docs/application/native/guides/app-management/ui-app/dali-app.md

```diff
@@ -0,0 +1,44 @@
+# DALi Applications
+
+
+You can use the Dynamic Animation Library (DALi) 3D UI toolkit to create a high-performance, rich UI application. For 2D-based Tizen native applications, use the [Enlightenment Foundation Libraries (EFL) UI toolkit](../../ui/efl/introduction-n.md).
+
+Dynamic Animation Library (DALi) internally defines a virtual 3D world (space) and maintains hierarchical objects in the 3D world. The hierarchical object tree is known as the [scene graph](http://en.wikipedia.org/wiki/Scene_graph).
+
+A node in the scene graph can have several children but often only a single parent, with the effect of a parent applied to all its child nodes; an operation performed on a group automatically propagates its effect to all of its members. There are various types of nodes, such as image, text, and buttons.
+
+## Basic Fundamentals
+
+Before you start UI programming with DALi, you must familiarize yourself with the basic DALi fundamentals. DALi consists of the following modules:
+
+- **DALi Core**
+
+  This module provides core functionalities, such as scene graph-based rendering, animation, and event handling. It is a base module and forms the biggest part of DALi.
+
+- **DALi Adaptor**
+
+  This module is a platform adaptation layer. It initializes and sets up DALi appropriately and provides many platform-related services with its internal module, platform abstraction. Several signals can be connected to it to keep you informed when certain platform-related activities occur.
+
+- **DALi Toolkit**
+
+  This module provides UI components and various effects on top of the DALi core.
+
+For more information on how to implement a 3D world using DALi, see [DALi Overview](../../ui/dali/dali-overview.md). It helps you to understand how to use DALi easily, while providing a Hello World example of constructing a virtual 3D world.
```

Comment:
The link is broken. It looks like `../../ui/dali/index.md` is correct.

Reply:
done